app/libs/commonCrudOperations.js

```diff
@@ -40,6 +40,19 @@ export const getById = (Model, findBy) => {
   };
 };
 
+export const removeById = (Model, findBy) => {
+  return (req, res, next) => {
+    const select = findBy
```

Comment:
you might see that in getById method we are using the same logic for selections. Please make this logic general.